content/module-reference/utility-modules/shared/collection-filters.md

```diff
@@ -0,0 +1,141 @@
+
+# module controls
+
+## adding new filters
+
+To add a new filter to the module, click on the "new rule" button and select an image attribute to use.
+
+## combining multiple filters
+
+When you use multiple filters, the first item on the filter header allows you to define how the filter is combined with the previous one.
+
+and
+: Narrow down search. An image is only retained if it also fulfills the new criteria.
+
+or
+: Add more images. Images from the _collection_ that fulfill the new criteria are added.
+
+and not
+: Exclude images. Images that fulfill the new criteria are removed.
+
+---
+
+**Note**: The and/or/not operators have a defined order of precedence such that "not" is executed first, then "and" and finally "or". This means, for example, that if you define complex filters like "A and B or C and not D" these will be implemented as "(A and B) or (C and (not D))".
+
+---
+
+## removing or deactivating filters
+
+You can remove or temporarily deactivate a specific filter using the buttons on the right of the filter header (see screenshots in the following sections). If a filter is pinned to the top panel, you will need to unpin it before removing or deactivating.
+
+## pinning into the top toolbar
+
+The pin button on the right of the filter header allows you to pin a filter to the top panel. To avoid unwanted actions, pinned filters can't be removed or deactivated.
+
+## resetting filters
+
+Clicking on the module reset button will remove all un-pinned filters and reset all others to their default values. If you want to also remove the pinned filters, you can Ctrl+click on the reset button.
+
+## returning to a previous set of filters
+
+You can return to a previously-defined set of filters by clicking on the history button at the bottom of the module and selecting from the resulting list.
+
+# filter widgets
+
+A number of filter widgets have been created for use within this module. Since some of these widgets use non-standard interfaces, their usage is explained in the following sections:
+
+## color labels
+
+The following image shows the color labels widget, set to filter images having yellow or green color labels:
+
+![color filter](./collection-filters/color-filter.png#w33)
+
+You can interact with this filter widget as follows:
+
+- Click on a color label to include images with that label
+- Ctrl+click on a color label to include images without that label
+- Click or Ctrl+click on the gray icon to act on all color labels simultaneously
+- Use the last button to define how multiple to handle the selection of multiple color labels. Select ![and](./collection-filters/color-filter-and.png#icon) (and) to filter images having _all_ of the selected color labels; Select ![or](./collection-filters/color-filter-or.png#icon) (or) to filter images with _at least one_ of the selected color labels.
```

Comment:
how ~~multiple~~ to handle the selection of multiple ...

Reply:
Well spotted thanks